docs/TextStyling.md

```diff
@@ -205,3 +205,21 @@ pdf.write_html("""<B>bold</B>
                   <B><I><U>all at once!</U></I></B>"""
 )
 ```
+
+
+## skew_text ##
+
+`skew_text` creates text that is skewed on the x or y axes:
+
+```python
+with pdf.skew_text(0, 10):
```

Comment:
Great doc!
Could you pass the parameters by name in this code sample, to be more explicit, please?

Reply:
sure, ill fix that